actions/extractor/codeql-extractor.yml

```diff
@@ -8,7 +8,8 @@ build_modes:
   - none
 file_coverage_languages: []
 github_api_languages: []
```

Comment:
Is it worth putting a comment here that `actions` isn't its own language as far as linguist/the API is concerned and therefore not listed?